tpm2_pytss/types.py

```diff
@@ -1100,6 +1100,10 @@ def __str__(self):
         b = self.__bytes__()
         return binascii.hexlify(b).decode()
 
+    def __eq__(self, value):
+        b = self.__bytes__()
```

Comment:
what if value is another instance of a TPM2B_SIMPLE_OBJECT? Can we just bytes(value) unconditionally to pick up comparisons between those and other "bytes like" objects?

Reply:
I just checked it seems that ```__eq__``` for bytes does that for us:
```Python
>>> from tpm2_pytss.types import *
>>> d = TPM2B_DIGEST("kaka")
>>> s = TPM2B_DIGEST("baka")
>>> s == d
False
>>> s = TPM2B_DIGEST("kaka")
>>> s == d
True
>>> 
``` 

But I'll add a check for it in the test as well